jira.go

```diff
@@ -54,12 +54,16 @@ func getJiraTickets(Mf MandatoryFlags, projectID string, customDebug debug) (map
 	responseData, err := makeSnykAPIRequest("GET", Mf.endpointAPI+"/v1/org/"+Mf.orgID+"/project/"+projectID+"/jira-issues", Mf.apiToken, nil, customDebug)
 	if err != nil {
 		customDebug.Debug("*** ERROR *** Could not get the Jira issues via Snyk API")
+		message := fmt.Sprintf("Could not get the tickets %s\n", err.Error())
+		writeErrorFile("getJiraTickets", message, customDebug)
 		return nil, errors.New("Could not get the tickets")
 	}
 
 	tickets, err := jsn.NewJson(responseData)
 	if err != nil {
 		customDebug.Debug("*** ERROR *** Could not read Jira issues via Snyk API")
+		message := fmt.Sprintf("Could not read Jira issues via Snyk API %s\n", err.Error())
```

Comment:
I can't see any logs for when we choose to skip creating jira tickets based on passed in options like skip issues that are not fixable? 

Can the messages be added for all issues that exist but we chose to skip making a ticket for it aka it is not an error it is a decision that is right now not visible to users.